tests/util.py

```diff
@@ -12,7 +12,7 @@
 
 from semantic_release.changelog.context import make_changelog_context
 from semantic_release.changelog.release_history import ReleaseHistory
-from semantic_release.cli import config as cliConfigModule
+from semantic_release.cli import config as cliConfigModule  # noqa: N812
```

Comment:
same feedback as other tests file - could rename it if there's no reason it has to be camel case here.